runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -948,8 +948,8 @@
 	int getMouseIdleTicks();
 
 	/**
-	 * Gets the time at which the last mouse press occurred in milliseconds since
-	 * 1st January 1970 00:00
+	 * Gets the time at which the last mouse press occurred in UNIX time measured in
```

Comment:
I would've called it "milliseconds since the UNIX epoch".

Reply:
There we go